src/uu/dd/cspell.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
It looks like CI is not picking this file up. We usually put the `words` as ignores on each file separately. Alternatively, you could add all of these words to `.vscode/cspell.dictionaries/jargon.wordlist.txt`. You can add the `ignorePaths` to the global `.vscode/cspell.json` file

Reply:
Ha! I didn't think to look in the .vscode folder for the spell checker overrides. I've added some dd jargon to the jargon file and an exception for the unit-tests. Thanks for pointing this out.